packages/react/src/patterns/sub-patterns/Card/README.md

```diff
@@ -41,7 +41,7 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | ----------- | -------- | --------- | ------------- | ---------------------------------------------------------------------------- |
 | `copy`      | NO       | String    | null          | Paragraph of text that further describing the resource with added detail.    |
 | `className` | NO       | String    | null          | Classname to be assigned to the Card component                               |
-| `title`     | YES      | String    | n/a           | Concise yet descriptive string of text describing the linked resource.       |
+| `title`     | NO       | String    | n/a           | Concise yet descriptive string of text describing the linked resource.       |
```

Comment:
@photodow  Will make note to address for that component.

Reply:
So I feel like this one is still relevant to this pull request since she changed `title` to `heading`. Need to update the documentation to reflect the change I believe.

Reply:
done